features/routing/haproxy-router.feature

```diff
@@ -352,12 +347,17 @@ Feature: Testing haproxy router
       | /tmp/cookies |
     Then the step should succeed
     And the output should contain "Hello-OpenShift"
+    Given I run the steps 10 times:
+    """
     When I execute on the pod:
-      | bash | -c | for i in {1..10} ; do curl -sS  http://<%= route("service-unsecure", service("service-unsecure")).dns(by: user) %>/ -b /tmp/cookies ; done |
```

Comment:
Don't remove the loop `for i in {1..10`}, it is required for this case.
Since bash is required for the loop and curl, so please don't update this case.

Reply:
Sure. But the bash doesn't work on the test-client-pod, maybe another PR is needed for this test case.

Reply:
just please keep the original pod-for-ping for this kind of cases, no need to update them for now.

Reply:
Try,
1. Replace `bash -c` with `sh -c`.
2. Remove `bash -c`

Reply:
@lihongan Yes, I keep the pod-for-ping for the bash. And for the first curl request before the bash, I use the test-client-pod. I also remove test-client-pod before creating pod-for-ping.

@liangxia Thanks. I tried sh -c yesterday, but it didn't work with the full command "sh -c for i in {1..10} ; ...".

Reply:
{1..10} is the expansion from bash. try `for (( i=1; i<=10; ++i ))`

Reply:
We're planning to rebuild the nginx-alpine image and to install more binaries/packages and enable more functions on it, so I prefer to update this case later with the new image that `bash` is built-in.

Reply:
@lihongan Thanks for the review and the new plan, I have rolled back case OCP-11679 to the original.